src/app/core/_components/forms/simple-forms/formconfig.component.html

```diff
@@ -0,0 +1,11 @@
+<horizontalnav [menuItems]="menuItems"></horizontalnav>
```

Comment:
Same as the previous templates, a mix of [attribute]="''" and attribute="" (I would prefer the last)

Reply:
Im binding here

Reply:
I meant the template in general